schematools/cli.py

```diff
@@ -88,6 +117,69 @@ def show():
     pass
 
 
+@schema.group()
+def permissions():
+    """Show existing metadata"""
+    pass
+
+
+@permissions.command("introspect")
+@option_db_url
+@argument_role
+def permissions_introspect(db_url, role):
+    """Retrieve ACLs from a database."""
+    engine = _get_engine(db_url)
+    introspect_permissions(engine, role)
+
+
+@permissions.command("revoke")
+@option_db_url
+@argument_role
+def permissions_revoke(db_url, role):
+    """Revoke all table select priviliges for role."""
+    engine = _get_engine(db_url)
+    revoke_permissions(engine, role)
+
+
+@permissions.command("apply")
+@option_db_url
+@option_schema_url
+@option_profile_url
+@argument_schema_location
+@argument_profile_location
+@argument_role
+@argument_scope
+@click.option("--dry-run", is_flag=True, default=False, help="Don't execute the GRANT statements")
+def permissions_apply(db_url, schema_url, profile_url, schema_location, profile_location, role, scope, dry_run):
+    """Set permissions for a postgres role associated with a scope from Amsterdam Schema or Profiles."""
+    def _fetch_json(location):
```

Comment:
Je zou jouw _fetch_json ook kunnen vervangen door deze bestaande functie:

`data = schema_fetch_url_file(schema_location)`

Dit is een functie die hetzelfde doet. Maar had ik eerder moeten zeggen! Dus kan ook een andere keer